project_stock/models/stock_move.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import _, api, exceptions, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    task_id = fields.Many2one(
+        comodel_name="project.task",
+        string="Task for finished products",
+        check_company=True,
+    )
+    raw_material_task_id = fields.Many2one(
+        comodel_name="project.task", string="Task for material", check_company=True
+    )
+    is_done = fields.Boolean(
+        string="Done",
+        compute="_compute_is_done",
+        store=True,
+        help="Technical Field to order moves",
+    )
+
+    @api.depends("state")
+    def _compute_is_done(self):
```

Comment:
Why do you need this field? The condition used for this computed field is easily reproducible on views.

Reply:
It is only necessary to order records in tree view.

Reply:
I think that feature is not needed.